pkg/printers/internalversion/describe_test.go

```diff
@@ -1664,9 +1664,25 @@ Spec:
     From Pod Selector: id=app2,id2=app3
     From Namespace Selector: id=app2,id2=app3
     From Namespace Selector: foo in (bar1,bar2),id=app2,id2=app3
+    From IPBlock:
+        CIDR: 192.168.0.0/16
+        Except: 192.168.3.0/24, 192.168.4.0/24
     ----------
     To Port: <any> (traffic allowed to all ports)
     From: <any> (traffic not restricted by source)
+  Allowing egress traffic:
+    To Port: 80/TCP
+    To Port: 82/TCP
+    To Pod Selector: id=app2,id2=app3
+    To Namespace Selector: id=app2,id2=app3
+    To Namespace Selector: foo in (bar1,bar2),id=app2,id2=app3
```

Comment:
Just noticed that there are two namespaceSelectors here. Is this by design or should one of these be PodSelector for the Peer type?

Reply:
@xiangpengzhao Did you note this feedback? Why are there two NamespaceSelector types? @caseydavenport Is this expected? I assumed one of these instances might be describing the PodSelector type in the NetworkPolicyPeer.

Reply:
Yeah I replied in the main thread :) https://github.com/kubernetes/kubernetes/pull/54853#issuecomment-342068974

Reply:
The reason why two NamespaceSelector types are here is because the test case has two different NetworkPolicyPeers both have NamespaceSelector types and they are overlapped. I'm not sure this case is allowed. If not, I guess we may have a bug and should.validate this case.

Reply:
I suggest we push this PR forward (I will modify the test case to avoid confusing if needed) and create a new isse to discuss this case :)

Reply:
I think this is the correct representation of this selector: https://github.com/kubernetes/kubernetes/pull/54853/files#diff-57ae6541425474ba84bed93a29b50281R1777

Reply:
ok that makes sense, thanks!